apps/server/src/modules/tool/service/external-tool.service.ts

```diff
@@ -24,6 +26,11 @@ export class ExternalToolService {
 		return duplicate == null;
 	}
 
+	async findExternalTool(query: Partial<ExternalToolDO>, options: IFindOptions<ExternalToolDO>) {
```

Comment:
This will find a list of external tools right? Why not call the method `findExternalTools` then or something else that describes that this can be many?

Reply:
Correct. I changed the name and added the missing return type